general-cookbook-guide.md

```diff
@@ -0,0 +1,107 @@
+# Chef Partner Cookbook Program
+
+## Who should write a Chef Partner Cookbook?
+This document is a specification for the Chef's Partner Cookbook Program. This is
+a general guideline taken from the [technical specification][cookbook_guide] and
+condensed down to these general points.
+
+If you have any questions or thoughts, please reach out to <partnereng@chef.io>.
+
+## What is a Chef Partner Cookbook?
+* A cookbook written by a vendor with guidance from Partner Engineering at Chef
+* A higher level of code quality compared to generic Community Cookbooks
+* A vendor ongoing support of the cookbook for their application or integration
+* A verified cookbook by both Chef and the vendor of the application or integration
+
+## Why Write a Chef Partner Cookbook?
+
+### Why Do People Write Chef Community Cookbooks?
```

Comment:
"Provide a consistent installation and configuration interface using a proven automation solution, like Chef"